### ISSUE TITLE
How to change streaming server. 

### ISSUE THREAD

**Issue by Ashotych** [ISBX/apprtc-ios#110]
Hello. I want to use this project for writing streaming application and want to undertand how to change streaming server. I opened test-audio room in Janus WebRTC Gateway. And if I have url with audio-room like **********************/audiobridgetest.html, can I use it in this example project? What should I do - just change server host url and skip part with creation of room? 
 Thank you for your attention.

**Comment by socquique** [comment]
Did you achieve to connect to the Janus WebRTC Gateway?

**Comment by Ashotych** [comment]
Yes, it works. But im not sure this lib or another. But webrtc voice streaming can be used in iOS with native.
